package.json

```diff
@@ -1,51 +1,51 @@
 {
-  "name": "ranvier",
-  "description": "MUD game engine",
-  "homepage": "https://ranviermud.com",
-  "bugs": "https://github.com/Ranvier-TS/core-ts/issues",
-  "license": "MIT",
-  "author": "Shawn Biddle (http://shawnbiddle.com), Sean O'Donohue, Brian Nelson",
-  "version": "3.1.1",
-  "repository": "github:ranvier-ts/core-ts",
-  "engines": {
-    "node": ">= 12.18.2"
-  },
-  "scripts": {
-    "test": "nyc --reporter=html --reporter=text --include=src --all mocha --recursive",
-    "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "prepare": "npm run build",
-    "build": "tsc",
-    "project": "npx tsc -p tsconfig.json",
-    "watch": "tsc --watch --project tsconfig.json",
-    "prepublish": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json"
-  },
-  "main": "./dist/cjs/index.js",
-  "types": "./dist/index.d.ts",
-  "module": "./dist/esm/index.js",
-  "directory": [
-    "./dist/esm"
-  ],
-  "dependencies": {
-    "@types/bcryptjs": "^2.4.2",
-    "@types/commander": "^2.12.2",
-    "@types/js-yaml": "^3.12.5",
-    "@types/uuid": "^8.3.0",
-    "@types/winston": "^2.4.4",
-    "@types/ws": "^7.4.5",
-    "bcryptjs": "^2.4.0",
-    "js-yaml": "^3.12.0",
-    "pretty-error": "^2.0.2",
-    "require-dir": "^1.1.0",
-    "sty": "",
-    "uuid": "^3.3.2",
-    "winston": "^2.4.5",
-    "wrap-ansi": "^2.0.0"
-  },
-  "devDependencies": {
-    "@types/node": "^14.14.2",
-    "coveralls": "^3.0.2",
-    "mocha": "^5.2.0",
-    "nyc": "^13.1.0",
-    "typescript": "^4.4.4"
-  }
+	"name": "ranvier",
+	"description": "MUD game engine",
+	"homepage": "https://ranviermud.com",
+	"bugs": "https://github.com/Ranvier-TS/core-ts/issues",
+	"license": "MIT",
+	"author": "Shawn Biddle (http://shawnbiddle.com), Sean O'Donohue, Brian Nelson",
+	"version": "3.1.1",
+	"repository": "github:ranvier-ts/core-ts",
+	"engines": {
+		"node": ">= 12.18.2"
+	},
+	"scripts": {
+		"test": "nyc --reporter=html --reporter=text --include=src --all mocha --recursive",
+		"coverage": "nyc report --reporter=text-lcov | coveralls",
+		"prepare": "npm run build",
+		"build": "tsc",
+		"project": "npx tsc -p tsconfig.json",
+		"watch": "tsc --watch --project tsconfig.json",
+		"prepublish": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json"
+	},
+	"main": "./dist/cjs/index.js",
+	"types": "./dist/index.d.ts",
+	"module": "./dist/esm/index.js",
+	"directory": [
+		"./dist/esm"
+	],
+	"dependencies": {
+		"@types/bcryptjs": "^2.4.2",
+		"@types/js-yaml": "^3.12.5",
+		"@types/uuid": "^8.3.0",
+		"@types/winston": "^2.4.4",
+		"@types/ws": "^7.4.5",
+		"bcryptjs": "^2.4.0",
+		"commander": "^4.1.0",
```

Comment:
had a compile error from commander in the node_modules. We only use the 'types' in core-ts, so just upgraded to the real package (as the types are just a stub) until the error went away.